source/enumerate-collections.rst

```diff
@@ -265,7 +266,17 @@ Example return::
         "system.replset"
     ]
 
-Drivers MUST strip the database name from the returned collection names.
+
+
+Drivers MAY implement a MongoClient method that returns an Iterable of strings,
```

Comment:
`listCollections` would be a method on the Database class. https://github.com/mongodb/specifications/blob/master/source/enumerate-collections.rst#driver-methods already mentions that all of the methods in this spec "should be on the database class".

If you're going to use "Iterable" here, you should probably copy over the term definition from https://github.com/mongodb/specifications/blob/master/source/enumerate-databases.rst#terms (and any other relevant terms, such as "MongoDatabase"). You may also want to add "MongoCollection" if a later section of this spec is going to talk about a method that returns an iterable of those objects.